### DISCUSSION TITLE
Motioneye in HA Core

### DISCUSSION THREAD

**Discussion by Dregi56** [indomus/forum#74] [HUB: Home Assistant]
Guardando la documentazione, trovo Motioneye come add-on integrabile su Hassio, ma non trovo nulla per Core.
Come si fa?

**Comment by snakuzzo** [comment]
Guarda...ti inviterei innanzi tutto a cercare sul sito
Inoltre ti chiederei la cortesia di non creare topic inutili tipo "come si fa a cancellare?" grazie

**Reply by snakuzzo** [reply]
@Dregi56 leggi...

https://indomus.it/progetti/realizzare-un-nvr-network-video-recorder-per-videosorveglianza-con-motioneye-parte-1/

**Reply by Dregi56** [reply]
Scusate.  :(

**Reply by Dregi56** [reply]
Ho letto nel sito la pagina dedicata. Io ho HA Core installato su SO Raspbian, ho seguito la strada di installazione di Motioneye con Doker. Con http://ipraspberry:8765 mi sono loggato ed ho aggiunto e visualizzato un camera IP. Ora come la visualizzo in HA Core?

**Reply by snakuzzo** [reply]
Eh...dici "scusate" però continui a non cercare sul sito :D
